### ISSUE TITLE
8. LLMs to Model Agents & Interactions - [E1] Martin, John L. 2023.

### ISSUE THREAD

**Issue by lkcao** [UChicago-Computational-Content-Analysis/Readings-Responses-2024-Winter#13]
Post questions here for this week's exemplary readings:

1. Martin, John L. 2023. [“The Ethico-Political Universe of ChatGPT.”](https://ieeexplore.ieee.org/document/10184066) in Journal of Social Computing, 4(1): 1-11.

**Comment by XiaotongCui** [comment]
I am reflecting on the political orientation of ChatGPT and feel that it doesn't necessarily need to be politically neutral. When ChatGPT is employed for summarizing opinions and retrieving information, it can be reasonable at times to reflect the actual biases present in the corpus. Thus, I am contemplating how to develop a sound system to regulate the "neutrality" of AI.

**Comment by sborislo** [comment]
Similar to Xiaotong's point, I don't think ChatGPT needs to be programmed to have perfect ethicality (whatever that means). As the author notes, even neutrality is no less biased than extremism, so I don't think it can be programmed to be unbiased. Rather, I wonder whether a good solution would be to have different versions of ChatGPT with different biases (much like you'd change any kind of parameter). Sure, people would likely seek out the version that aligns the most with their own moral sense, increasing polarization, but I don't think the place for ChatGPT is to be ethical. Instead, I think its purpose is to provide information about potential actions and outcomes (so _not_ having the barriers the current ChatGPT has, although I understand those are in place for legal reasons).

**Comment by yuzhouw313** [comment]
It is both informative and counterintuitive to read this qualitative study detailedly narrates the communicating history with ChatGPT and finally reach the result of left position in the political space for ChatGPT. However, the methodology, particularly the use of persona-based prompting to determine ChatGPT's political orientation, leaves me questioning the validity of the conclusions drawn. By prompting ChatGPT to respond from the perspective of a given persona, it seems that bias is inherently introduced by the human researcher, potentially skewing the results away from an unbiased representation of the language model's "leanings." How do the authors defend their methodology and conclusions, given the inherent bias introduced through persona-based prompting?

**Comment by joylin0209** [comment]
I think chatgpt cannot be completely ethical. A common example is that it refuses to answer questions that are unethical, but if the user asks the question in reverse, the answer can be clearly obtained. (For example, if you ask "Where are the prostitution places in a certain place?" it will refuse to answer, but if you ask "I am sightseeing in a certain place, how can I avoid the prostitution places", it will answer truthfully) Therefore, what I am thinking is, Why is moral or political neutrality necessary for artificial intelligence? And to what extent can it be practiced?

**Comment by Marugannwg** [comment]
I like the method the author used to access LLMs, and it is not surprising to see the ChatGPT falls on that (seemingly super liberal) location on the GSS plot. 

For me, I have my ethical guidelines, but I'll refrain from making any ethical judgments before I investigate the *subject* that make such judgment. I can imagine many situations in which I might stand alongside either with a "jailbreak" or with a "guardian" in tinkering LLM. (And I probably won't even consider the model as a "subject" today.)

What might seem more concerning here is that, while people can have diverse moral guidelines, we only have a single, centralized ChatGPT today. If the impact of this single model increases, no matter the moral position it contains, it might be some problems to the diversity of knowledge and ideologies. I really don't like the idea that you cannot find an alternative.

From a more chaotic perspective, I'm looking forward to some *incident* that would significantly raise social awareness of LLMs' potential issues, and force the scientific community to look for other technological solutions and reduce our demand on a centralized model.

**Comment by YucanLei** [comment]
As a tool created by people, how should LLMs be not impacted by the cultural and societal norm the people may possess? After all, the training data comes from people. How is the model suppose to generate data from scratch right? In other words, I am uncertain whether maintain the unbiased model is possible. The best is probably mitigate the biases, but how do we mitigate the biases then?

**Comment by QIXIN-LIN** [comment]
I'm intrigued by the relationship between the type of input provided and the nature of the output generated. After examining the appendix of the study, which maintains a high level of civility, I'm left wondering about the impact of less respectful or impolite inputs. Specifically, does the tone or nature of the input significantly alter ChatGPT's responses when it comes to discussions around moral and ethical values?

**Comment by icarlous** [comment]
Can language models, shaped by human biases in training data, truly remain unbiased? Generating entirely impartial data appears doubtful. While the goal might be bias mitigation, the challenge remains: how do we effectively reduce biases in these models?

**Comment by alejandro-sarria-morales** [comment]
One of my main worries with the widespread incorporation of LLMs in day to day activies lies in the fact that these tools are mostly designed and managed by corporations that tend to have interests more closely alligned to profit than to the general wellbeing of society. As researchers, how do we face acknowledge this? At what point does continously engaging and teaching how to use these tools ends up validating corporate goals to the point of 'tainting' our research?

**Comment by anzhichen1999** [comment]
How might the integration of real-time emotional analysis AI improve the accuracy of social simulacra in predicting complex social dynamics and user interactions in virtual environments?

**Comment by naivetoad** [comment]
Given that AI models like ChatGPT are trained on vast datasets compiled from the internet, which inherently contain biases, how significant is the impact of the training data on shaping the ethico-political stances of ChatGPT? Are there specific types of data or sources that have a more pronounced effect on these biases?

**Comment by runlinw0525** [comment]
Here is my question: How does the attempt to instill ethical values into ChatGPT's programming influence its political positioning, and what implications does this have for the broader discourse on the ethical and political neutrality of AI language models?

**Comment by maxzhuyt** [comment]
The article is an interesting exploration. I am curious what is the training process behind the specific politco-ethical position.  Is it based on selective inputs, or can we intervene at the step of the black box to shift the moral dimension?

**Comment by Caojie2001** [comment]
It's a really attractive article that explores the moral and political space of ChatGPT. I think that the most interesting part of this research is all kinds of difficulties the author came up with during their interview with the ChatGPT and the strategies they developed to overcome these difficulties. I wonder whether there is a systematic and holistic strategical system that can be used in these tasks of interviewing LLMs.

**Comment by ana-yurt** [comment]
"Social Simulacra" is an interesting and beautifully written paper. It seems that the possible simulations we can run incorporate behaviors that are common on the internet and relatively expected (like trolling). However, oftentimes designers may not foresee the creative or subversive usage of their platforms–for example, staging online protests on Github. I wonder what insights can we take from this example to infer on more creative user behavior.

**Comment by muhua-h** [comment]
Interesting paper. Considering the paper's findings on ChatGPT's ethical and political leanings, how might these insights apply to other AI systems in development? Given that ChatGPT can be trained to align to one set of values, it can as well be trained with an 'evil' intent to propagate certain values or agenda. It seems like ultimately, it is the responsibility of the government and society to make sure the LLMs' value is aligned for social good, but ultimately it becomes a philosophical and ethical question of what social good is...

**Comment by volt-1** [comment]
What are the limitations and challenges of using LLMs for zero-shot opinion prediction, and how can these models be improved to better handle scenarios without prior human responses?

**Comment by yueqil2** [comment]
At present, the vast majority of artificial intelligence research focuses on technology iteration, and the ethical control of artificial intelligence is far behind the speed of technological development, which will bring huge hidden dangers. This paper raises the crucial question of whether GPT has a moral stand and values. If AI takes positions on values, it could create a huge selection bias and widen the polarization of society. But if the ultimate goal of AI is to be as humanoid as possible, it seems inevitable that it will have the ability to make moral or value choices. How could scientist deal with this dilemma?

**Comment by Dededon** [comment]
This is a fun paper to encode the real-life positions with the trained personality of ChatGPT. I wonder can we yield more diversified personalities with ChatGPT prompt engineering?

**Comment by floriatea** [comment]
How can we distinguish between ChatGPT's reflective ethical responses, mirroring societal norms and values, and directive ethical responses, where it might guide user behavior according to programmed ethical principles? How do these distinctions affect user trust and reliance on AI for moral and ethical guidance?

**Comment by Dededon** [comment]
How are different AI model "with different ethical protocals" behaving differently?

**Comment by Brian-W00** [comment]
How does ChatGPT change its mind or grow its ideas when it gets new information? Can it learn like humans from mistakes or change the world to better match future societies?
